### DISCUSSION TITLE
[Feature] ability to download photos in bulk from immich server to phone's internal storage

### DISCUSSION THREAD

**Discussion by hari-dev06** [immich-app/immich#14365] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think it would be great to have a feature like that, i used to run immich server for the photos on my old phone had a lot of pictures saved, bought a new phone and i could see the photos from my old phone only on immich and had no option to download all of them in bulk. Would be great to have this feature

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#5068
